content/en/docs/examples/microservices-istio/add-mtls/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Enable mutual TLS Authentication with Istio
+overview: Enable Mutual TLS Authentication on the traffic between microservices
+weight: 75
+
+---
+
+In your current configuration, microservices communicate using HTTP. The traffic
+is not encrypted and if your cluster is compromised, the attackers can eavesdrop
+on the traffic. In addition, in some environments, there are regulations that
+require that all the traffic inside the cluster must be encrypted.
+
+Istio helps solve this problem by encrypting the traffic between the
+sidecars. This leaves only the traffic inside the application pods, and traffic
```

Comment:
I fixed the previous comment removing the threat model comments. I'm leaving this here because it's factual and I'd want to know as a reader. But I'm open if someone has specific words they want to say here instead to address this.